src/tests/components/Alert.test.js

```diff
@@ -0,0 +1,11 @@
+import { describe } from "@jest/globals";
+import React from "react";
+import { shallow } from "enzyme";
+import Alert from "../../components/commons/Alert";
+
+describe("<Alert/>", () => {
+  it("Should render alert correctly", () => {
+    const wrapper = shallow(<Alert message="" isOpen={true} />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
please avoiding using toMatchSnapshot, [read this for reference ](https://medium.com/@tomgold_48918/why-i-stopped-using-snapshot-testing-with-jest-3279fe41ffb2)